db/migrate/20150720142931_reregister_archived_whitehall_artefact_routes.rb

```diff
@@ -0,0 +1,12 @@
+class ReregisterArchivedWhitehallArtefactRoutes < Mongoid::Migration
+  def self.up
+    Artefact.where(:state => 'archived', :owning_app => 'whitehall').each do |artefact|
+      # register the routes with the router.
+      RoutableArtefact.new(artefact).register
+    end
+    RoutableArtefact.new(nil).commit
```

Comment:
It's the easiest way to ask `RoutableArtefact` to call `commit` at the router. It should perhaps be a class method as it doesn't rely on the state of any `RouteableArtefact` object.